build.xml

```diff
@@ -173,6 +173,7 @@ java -cp ${jar.absolute.path}:${lib.classpath} kbasesearchengine.tools.SearchToo
             <include name="kbasesearchengine/test/events/storage/MongoDBStatusEventStorageTest.java"/>
             <include name="kbasesearchengine/test/integration/IndexerIntegrationTest.java"/>
             <include name="kbasesearchengine/test/integration/IndexerWorkerIntegrationTest.java"/>
+            <include name="kbasesearchengine/test/parse/ObjectParserTest.java"/>
```

Comment:
Minor - the first line of the fileset has kbasesearchengine/test/parse/**Test.java. We should either remove this line or remove the first line and add all parse package tests individually to keep with the theme.

Reply:
I've completely redone the test target locally.

Reply:
Seeing as there are only two test files, I'll go for the latter option